### DISCUSSION TITLE
100W+ Water ERT Module  ERW-1300-402

### DISCUSSION THREAD

**Discussion by garyh2626** [bemasher/rtlamr#264] [General]
Thanks for all the work on RTLAMR, i am successfully reading both my electric and gas meter.
Now I am trying to get this working with a recently installed Water module. RTLAMR is receiving my meter since the endpoint number matches the number on my meter. reading in json mode: Protocaol: 0x1E, EndpointType: 0xAB, Endpoint: MyMeterID. Consumption: 126

The problem is consumption only shows first 3 number of the reading and missing the last 5 numbers including decimal point.  126 where the meter actually reads 12652.111

**Comment by kb1ca** [comment]
Although I am not familiar with your specific water meter I can say that it's quite common for AMR transmissions to truncate some of the least significant register digits if they're not considered consequential for billing purposes. Eg. If your supplier effectively reads 12600.000 over the air this month  and let's say 12700.000 next month then you'll still end up paying for the 52.111 that wasn't read out today.

As for why, water meters are usually entirely battery powered and every digit that is transmitted consumes a tiny bit of that finite energy reserve. With batteries designed to ideally last tens of years every transmitted bit becomes consequential.

**Reply by garyh2626** [reply]
That does make some sense but not exactly since the meter is continuously spitting readings every minute or two for the expected 10 year battery. I will have to wait for my next bill to arrive but if I recall correctly they only bill by 100 gallon increments and may another factor to only transmit the significant digits.

My meter is extremely accurate down to 1/1000 of a gallon and would be great to use that data as water leak detector.

**Comment by garyh2626** [comment]
Found previous water bill and confirmed the water company only charge for usage per CCF, 100 cubic feet = 748 gallons. From bill: previous reading 121, current reading: 125, units: 4, measurement CCF. This explains transmitting only significant digits for billing.

**Reply by hanley-development** [reply]
@garyh2626 I know this is almost a year late but do you have an example of one of your data samples?
trying to gather my data I am not seeing it transmit.

**Reply by andhab1** [reply]
I have a similar meter, ending in 403, but i've yet to receive any data from it. Could you share your command line arguments in case there's something i'm missing?

**Comment by subkron** [comment]
Maybe obvious (and this re3sponse is late to the game), but that ERT transmits on scm+.
